Main.py

```diff
@@ -392,15 +392,14 @@ def precollect_hint(location):
                     f.write(bytes([3]))  # version of format
                     f.write(multidata)
 
-            multidata_task = pool.submit(write_multidata)
+            output_file_futures.append(pool.submit(write_multidata))
             if not check_accessibility_task.result():
                 if not world.can_beat_game():
                     raise Exception("Game appears as unbeatable. Aborting.")
                 else:
                     logger.warning("Location Accessibility requirements not fulfilled.")
 
             # retrieve exceptions via .result() if they occurred.
-            multidata_task.result()
```

Comment:
I was thinking that comment was still appropriate, because that's what the loop below it is doing.

The logging will report an extra number, because of the included multidata, but that's also not really a problem, because that's an output file, right?

(It looks like the number already didn't match the number of players, because it included `call_stage` with all the `call_single`.)

Reply:
correct, the comment was intentionally left there, as it's still appropriate.